mappings/net/minecraft/resource/featuretoggle/FeatureFlags.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_7701 net/minecraft/resource/featuretoggle/FeatureFlags
+	FIELD field_40177 VANILLA Lnet/minecraft/class_7696;
+	FIELD field_40178 BUNDLE Lnet/minecraft/class_7696;
+	FIELD field_40179 UPDATE_1_20 Lnet/minecraft/class_7696;
+	FIELD field_40180 FEATURE_MANAGER Lnet/minecraft/class_7697;
+	FIELD field_40181 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_40182 VANILLA_FEATURES Lnet/minecraft/class_7699;
+	FIELD field_40183 VANILLA_FEATURES2 Lnet/minecraft/class_7699;
```

Comment:
Ideally, two fields of the same value should be named from their usages.

Reply:
I don't know how to see the usages. They are the exact same in `FeatureFlags`.

Reply:
In Enigma, right-click the name, then click `Show Calls` on the menu.

Reply:
```suggestion
	FIELD field_40182 VANILLA_FEATURES Lnet/minecraft/class_7699;
	FIELD field_40183 DEFAULT_ENABLED_FEATURES Lnet/minecraft/class_7699;
```

The former is a property of a block etc, and the latter is the value used to indicate loading such values.